PluginConfig.cs

```diff
@@ -10,25 +10,37 @@
 
 using System.Runtime.CompilerServices;
 using IPA.Config.Stores;
+using UnityEngine;
 
 [assembly: InternalsVisibleTo(GeneratedStore.AssemblyVisibilityTarget)]
 namespace PerformanceMeter {
     public class PluginConfig {
         public static PluginConfig Instance { get; set; }
         public virtual bool enabled { get; set; } = true;
-        public virtual int mode { get; set; } = (int)MeasurementMode.Energy;
-        public virtual int side { get; set; } = (int)MeasurementSide.Both;
-        public virtual int secondaryMode { get; set; } = (int)MeasurementMode.None;
-        public virtual int secondarySide { get; set; } = (int)MeasurementSide.Both;
+        public virtual MeasurementMode mode { get; set; } = MeasurementMode.Energy;
+        public virtual MeasurementSide side { get; set; } = MeasurementSide.Both;
+        public virtual Color sideColor { get; set; } = Color.white;
```

Comment:
I don't quite understand the purpose of the `sideColor` setting. Why is it necessary to have a distinction between this and just the normal `color` override? If the user wants to change the color of each side, they can just change the normal `color` variable.